src/cpp/fastrtps_deprecated/subscriber/SubscriberHistory.cpp

```diff
@@ -580,10 +580,6 @@ std::pair<bool, SubscriberHistory::instance_info> SubscriberHistory::lookup_inst
     {
         if (handle.isDefined())
         {
-            if (exact)
```

Comment:
I suppose this function has been reviewed in other PR.
However, I would suggest to add comments to this function. At least specify in function header that a NO_KEY topic cannot be looked up with a handle or exact value.

Reply:
Done in 83579e4